CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### [0.13.0] - 2024-12-18
```

Comment:
Can leave this as `Unreleased`. this will be updated when we release this version!